pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=80", "setuptools-scm>=8" ]
+
+[project]
+name = "pymbar"
+description = "Python implementation of the multistate Bennett acceptance ratio (MBAR) method"
+readme = "README.md"
+keywords = [ "Bayesian parameterization", "forcefield", "molecular mechanics" ]
+license = "MIT"
+authors = [
+  { name = "Levi N. Naden", email = "levi.naden@choderalab.org" },
+  { name = "Jaime Rodriguez-Guerra", email = "jaime.rodriguez-guerra@choderalab.org" },
+  { name = "Michael R. Shirts", email = "michael.shirts@virginia.edu" },
+  { name = "John D. Chodera", email = "john.chodera@choderalab.org" },
+]
+requires-python = ">=3.10"
+classifiers = [
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
```

Comment:
```suggestion
```

Reply:
I wanted this to match the `requires-python = ">=3.10"` line (I didn't check to see if we were using any 3.10+ features but felt like it wasn't a bad floor to enforce)